docs/en/advanced-concepts/settings.md

```diff
@@ -67,7 +67,8 @@ Note that the actual commands you would use in your test script differ based on
 |`shutdownOnPowerDisconnect`| Shutdown the server through the broadcast receiver on [ACTION_POWER_DISCONNECTED](https://developer.android.com/reference/android/content/Intent.html#ACTION_POWER_DISCONNECTED). Defaults to `true` | `false` or `true` |
 |`trackScrollEvents`| Turn on or off the tracking of scroll events as they happen. If `true`, a field, `lastScrollData`, is added to the results of `getSession`, which can then be used to check on scroll progress. Turning this feature off significantly increases touch action performance. Defaults to `true` | `true` or `false` |
 |`wakeLockTimeout`| Allows to change the timeout of the acquired wake lock. The lock is acquired on server startup and is held until the UIAutomator2 server is killed or the timeout expires. Setting this value to zero or a negative number will release the lock immediately if it is held. Defaults to '24 * 60 * 60 * 1000' milliseconds. | e.g. `0`, `60000` (1 min) |
-|`enableMultiWindows`| Tries to get available active windows. It might help to resolve finding elements on windows like `android.widget.PopupWindow`. Defaults to `false` and it returns an active root window.  It requires API Level 21+. Read [appium-uiautomator2-server#301](https://github.com/appium/appium-uiautomator2-server/pull/301) for mroe details. |`true` or `false` |
+|`enableMultiWindows`| If enabled then the page source and xpath lookup will include all windows available to the accessibility rather than the active one only. It might help to resolve finding elements on windows like `android.widget.PopupWindow`. Defaults to `false` and returns an active root window.  It requires API Level 21+. Read [appium-uiautomator2-server#301](https://github.com/appium/appium-uiautomator2-server/pull/301) for mroe details. |`true` or `false` |
```

Comment:
`It requires API Level 21+.` might be skipped, since uia2 driver already contains a startup check, which verifies the API version